### ISSUE TITLE
resume non-existent changefeed, cli don't report any error

### ISSUE THREAD

**Issue by Tammyxia** [pingcap/tiflow#1865]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

- Resume a changefeed which not existed, 
 
![image](https://user-images.githubusercontent.com/78345569/119929215-aac2b080-bfaf-11eb-942f-353027dd1bc5.png)

2. What did you expect to see?

- Cli report error like:  There's no changefeed "replication-task"...

3. What did you see instead?

- It seems cli can resume non-existent cf.

4. Versions of the cluster

    - Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

        ```
        5.0.0-nightly 5-24-2021
        ```

    - TiCDC version (execute `cdc version`):

        ```
        5.0.0-nightly 5-24-2021
        ```

**Comment by hi-rustin** [comment]
I am working on this.

/assign

**Comment by hi-rustin** [comment]
I can reproduce it.

Currently, we ignore the changefeed non-existent error, see: https://github.com/pingcap/ticdc/blob/d933dd17544711f6fed721723270bb4408882d7e/cdc/http_handler.go #L305

Discussed offline, we may fix it when the API is rewritten in the next sprint.

cc: @leoppro

**Comment by asddongmen** [comment]
Close it since it is stale. It has been fix in > v6.1.0.
